etc/ex.dev.config.yaml

```diff
@@ -20,5 +20,5 @@ openshift:
 broker:
   dev_broker: true
   launch_apb_on_bind: false
+  recovery: true
```

Comment:
Can you please add this to the template too? templates/deploy-ansible-service-broker.yaml under configMap. https://github.com/openshift/ansible-service-broker/blob/master/templates/deploy-ansible-service-broker.template.yaml#L211